plugin/trino-delta-lake/src/main/java/io/trino/plugin/deltalake/procedure/VacuumProcedure.java

```diff
@@ -200,7 +202,12 @@ private void doVacuum(
             List<Long> recentVersions = transactionLogAccess.getPastTableVersions(fileSystem, transactionLogDir, threshold, tableSnapshot.getVersion());
             Set<String> retainedPaths;
             try (Stream<AddFileEntry> activeAddEntries = transactionLogAccess.getActiveFiles(
-                    session, tableSnapshot, handle.getMetadataEntry(), handle.getProtocolEntry())) {
+                    session,
+                    tableSnapshot,
+                    handle.getMetadataEntry(),
+                    handle.getProtocolEntry(),
+                    TupleDomain.all(),
+                    alwaysFalse())) {
```

Comment:
This was alwaysTrue() before. It this change intended? Maybe it should be in separate commit?